src/core/or/circuit_st.h

```diff
@@ -234,4 +239,9 @@ struct circuit_t {
   struct circpad_machine_runtime_t *padding_info[CIRCPAD_MAX_MACHINES];
 };
 
+/* Declare the handle helpers */
+HANDLE_DECL(circuit, circuit_t, )
```

Comment:
I think that these belong in one of the circuit*.h files -- we only put structures in _st.h files.